### DISCUSSION TITLE
Peppol Reporting Implementation

### DISCUSSION THREAD

**Discussion by dhayalcbe** [OxalisCommunity/oxalis#662] [General]
I am new developer using oxalis, and trying to use Peppol Reporting provided by Oxalis. Where should I start with to implement and send EUSR and TSR to Peppol? Is there any guideline or reference to follow? I am using V6.1.1 of oxalis

**Comment by DeepaZirius** [comment]
@aaron-kumar can you throw some light on how to use the implementation provided in Oxalis V6.1.1 ?

**Comment by aaron-kumar** [comment]
@dhayalcbe , @DeepaZirius , please refer: https://github.com/OxalisCommunity/oxalis/wiki/Peppol-Reporting
